bootstrap/bukkit/src/main/java/org/geysermc/platform/bukkit/world/GeyserBukkitWorldManager.java

```diff
@@ -42,4 +44,14 @@ public BlockState getBlockAt(GeyserSession session, int x, int y, int z) {
         }
         return BlockTranslator.getJavaIdBlockMap().get(Bukkit.getPlayer(session.getPlayerEntity().getUsername()).getWorld().getBlockAt(x, y, z).getBlockData().getAsString());
     }
+
+    @Override
+    public void setGameRule(GeyserSession session, String name, Object value) {
+        Bukkit.getPlayer(session.getPlayerEntity().getUsername()).getWorld().setGameRuleValue(name, (String) value);
```

Comment:
I actually have no idea about this line and i havent tested it on bukkit yet.

Reply:
This works once PR 490 is applied.